core/embed/projects/kernel/main.c

```diff
@@ -204,11 +204,12 @@ void drivers_init() {
 
 // defined in linker script
 extern uint32_t _codelen;
-extern uint32_t _coreapp_clear_ram_0_start;
-extern uint32_t _coreapp_clear_ram_0_size;
-extern uint32_t _coreapp_clear_ram_1_start;
-extern uint32_t _coreapp_clear_ram_1_size;
+
 #define KERNEL_SIZE (uint32_t) & _codelen
+#ifndef AUX2_RAM_START
```

Comment:
Perhaps instead of defining a fake `AUX2_RAM_START`, we could use the same approach as with `FIRMWARE_P2_START`:

```
#ifdef AUX2_RAM_START
      .data2.start = (uint32_t)AUX2_RAM_START,
      .data2.size = (uint32_t)AUX2_RAM_SIZE,
#endif
```

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/5e57d8d4626a6ad0dbd44ae630dd040781c64f13